Makefile

```diff
@@ -36,13 +50,19 @@ container-image: ## Build container image
 binary-snapshot: ## Build binary snapshot
 	goreleaser release --rm-dist --skip-publish --snapshot
 
+##@ Checks
+
 .PHONY: check
 check: test lint ## Run checks (tests and linters)
 
 .PHONY: test
 test: ## Run tests
 	go test -race -v ./...
 
+.PHONY: test-e2e
+test-e2e: ## Run e2e tests
+	bats e2e
```

Comment:
is there a way to install bats dependency along with the other ones (similar to `bin/licensei` by adding `bin/bats`)?